app/views/ChooseGiveReasonItemDamagedView.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    h1: components.h1
+)
+
+@(form: Form[_], mode: Mode)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("chooseGiveReasonItemDamaged.title"))) {
+
+    @formHelper(action = routes.ChooseGiveReasonItemDamagedController.onSubmit(request.ern, request.arc, mode), 'autoComplete -> "off") {
```

Comment:
It would be cleaner to pass the action into the view from the Controller. So instead of passing the `mode: Mode` in as a parameter pass in an `action: Call`.

We could update the scaffolds at somepoint to do this, so that we don't need to change.